understanding/20/keyboard.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Keyboard</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Everything can be done with a keyboard</dd>
+		<dt>Author task</dt><dd>Do not provide user actions that cannot be accomplished by keyboard</dd>
```

Comment:
@gundulaniemann wrote
> What about a positive wording like "Enable accomplishment by keyboard for all user actions."?

Open to this type of change, although as per prior comments, suggest the Objective (which here states "Everything can be done with a keyboard") be read in concert with the Author task. My experience is that the author challenge for keyboard is useful to express as I have. Welcome discussion.

Reply:
The double negative might be hard to understand for some readers. According to my experience, also users with no disability found have issues in processing a double negation.